### PR TITLE
first modifications to refactor the code

### PR DESCRIPTION
**Pull request by juliatogr**
## 🔗 Tasks
- Commented the temporal call to test functions
- organize parameters and returns, 
- organize paths and prints
- review melodia algorithm

## 👉 What?

- Commented the temporal call to test functions
- Every extractor asks for the audio data and sampling rate and returns the frequency
- Now path to record is the same as path to the sound to analyse
- Now there is only one print on the main instead of one on each extractor
- Deleted melodia algorithm

## 🤷 Why?
- testing calls were executed when importing files
- calling each extractor is now easier
- format between extractors is more similar
- melodia algorithm was unnecessary and caused a lot of problems

## ⚙️ How?
- deleted every print on the extractors and included it on the pitch_extractor function.
- Entering the audio and sr to every extractor lets read the file before calling them on the pitch_extractor function
- every 'data' variable is now 'audio' and every 'fs' or 'sampling_rate' is sr.

## 👁‍🗨 Testing?
Nothing

## 🏭 Refactoring?
Everything was refactoring.

## 📸 Screenshots (optional)
No screenshots

## 🤔 Anything Else?
I want to mantain the same names on the extractor and deeply analyse the structure of the code.

### CODE REVIEW

**Review by Mario229997 — APPROVED** [review]
👌😉

File: offline_pitch_extractor/crepe_extractor.py [hunk 1]
```diff
@@ -28,12 +28,12 @@ def crepe_pitch(audio, sr):
...
                 idxs.append(idx)
 
-    print("Detected frequency: ", frequency[idxs][0].round())
-
     # print("time: ", time[idxs], "frequency: ", frequency[idxs], "confidence: ", confidence[idxs])
     # print("time: ", time, "frequency: ", frequency, "confidence: ", confidence)
     # print(len(audio), "\n", len(time), "\n",  len(frequency), "\n",len(confidence),  "\n", len(activation))
-    return time, frequency, confidence, activation
+    #return time, frequency, confidence, activation
+    freq = frequency[idxs][0].round()
```
Comment (nereasastre) [inline offline_pitch_extractor/crepe_extractor.py:35]: mmm potser hauriem de retornar la mean? ja que potser la primera esta malament idk

Comment (nereasastre) [inline offline_pitch_extractor/crepe_extractor.py:35]: potser hauriem de retornar la mean? perque si la primera esta mal maybe es shit

**Review by nereasastre — APPROVED** [review]

File: offline_pitch_extractor/crepe_extractor.py (hunk 1 above)
Comment (juliatogr) [reply offline_pitch_extractor/crepe_extractor.py:35]: ho he canviat. Faig pull i merge directament, que tira be!

**Pull request merged**